src/content/en/2022/page-weight.md

```diff
@@ -4,19 +4,443 @@ title: Page Weight
+  sheets_gid="159262492",
+  sql_file="response_format_distribution.sql"
+  )
+}}
+
+Distribution of image sizes by formats shows us that JPG, WebP and PNG file formats retain their 2021 status as top sources of image weight.
+
+{{ figure_markup(
+  image="distribution-of-image-sizes.png",
+  caption="Image size distribution.",
+  description="Bar chart showing the distribution of image sizes by percentile. A desktop page loads 82 KB at the 10th percentile, 331 KB at the 25th percentile, 1,026 KB at the 50th percentile, 2,694 KB at the 75th percentile, 6,066 KB at the 90th percentile. A mobile page loads 64 KB at the 10th percentile, 260 KB at the 25th percentile, 881 KB at the 50th percentile, 2,402 KB at the 75th percentile, 5,474 KB at the 90th percentile",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vS-yCXlRH7TMaQg1T2YsESIiUNe4BSga5NWwEElHgU3ZaNLXNauwAxsn3k_el5v1fxvtc_YA5k9i689/pubchart?oid=692759206&format=interactive",
+  sheets_gid="1763112644",
+  sql_file="page_weight_trend.sql"
+  )
+}}
+
+The median desktop image weight for 2022 was 1,026 kilobytes, a mere 44 kilobyte increase from 2021. Mobile barely shifted at 881 kilobytes.
+
+The year over year consistency is only disrupted by the extremities of the 100th percentile. The largest desktop page contained 672MB of images, a significant increase from the hefty max of 186MB in 2021. Similarly, the mobile 100th percentile saw a 959% increase to 385MB.
+
+#### Video bytes
+
+According to the media chapter of the mobile web, 5% of mobile pages include the `video` element. This information aligns with the 100th percentile of other file type in overall page weight (as video files are grouped in the set). Pages bringing video experiences take on a corresponding increase in weight.
+
+MP4s, which represent [51.5% of videos](./media#video) on the web, also represent the capacity for largest response size. At the 50th percentile, mp4 response sizes come in at 534 kilobytes.
+
+{{ figure_markup(
+  image="distribution-of-video-sizes-by-format.png",
+  caption="Image size distribution.",
+  description="Box chart showing the distribution of image sizes by format. mp4 is the largest at the 50th percentile, with 534 KB, followed by f4v at 155 KB, flash at 257 KB, and flv at 720 KB.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vS-yCXlRH7TMaQg1T2YsESIiUNe4BSga5NWwEElHgU3ZaNLXNauwAxsn3k_el5v1fxvtc_YA5k9i689/pubchart?oid=1651375033&format=interactive",
+  sheets_gid="159262492",
+  sql_file="response_format_distribution.sql"
+  )
+}}
+
+## Adoption rates of byte-saving technologies
+
+So what can we do about all those bytes we're sending? Well obviously we could just stop sending them, but obviously they are being sent for a reason—you'd hope! So let's look at how to keep the content, but send it in more efficient ways than just stuffing kilobytes and kilobytes down the pipes.
```

Comment:
Added this as an intro to this section as we don't like "hanging headers" where we have an H2 immediately followed by an H3 with no text in between.